scripts/excelToJson.py

```diff
@@ -4,17 +4,19 @@
 from pathlib import Path
```

Comment:
These questions may have already been discussed on the mob call, so apologies if that is the case. 
1) Before this is merged, is everyone agreed on the JSON output (including feature teams)?
2) Should we also include the updated output JSON as part of this merge?
3) This script still appears to generate the individual files in the output_json directory - M1_flattened.json, N1_flattened.json etc. We then have to copy that across to the file that is used to seed the table (iva-defects.json).

Are we going to stick with these individual output files? (Maybe there is a reason this could be helpful with some of the other EU vehicle categories if they require different logic to parse).

However if the logic will be the same for all eu vehicle categories, is now a good opportunity to a) churn out all of the flattened defects into 1 file and b) point directly to the seed file rather than have duplicates?